apps/expo/src/components/Comment.tsx

```diff
@@ -42,14 +42,18 @@ export default function Comment({ comment }: { comment: CommentComponent }) {
           source={{
             uri: "https://static.wikia.nocookie.net/acc-official-database/images/9/91/El_gato.jpg/revision/latest?cb=20220709001857",
           }}
-          className="h-8 w-8 rounded-full"
+          className="h-12 w-12 rounded-full"
           resizeMode="cover"
         />
-        <Text className="ml-2 mr-2 text-base">{comment.userId}</Text>
-        <Text>Icon</Text>
-        <Text className="ml-2 text-base">{formatElapsed()}</Text>
+        <View>
+          <Text className="font-poppins400 ml-4 text-base">John Doe</Text>
```

Comment:
for text size, use the custom sizes that we created in tailwind: https://github.com/uoftblueprint/centre-for-dreams/pull/212

Reply:
ok